lib/geordi/gitlinear.rb

```diff
@@ -0,0 +1,201 @@
+      issues = applicable_issues
+      reset_loading_message = "\r#{' ' * (loading_message.length + issues.length)}\r"
+
+      Geordi::Interaction.fail('No issues to offer.') if issues.empty?
+
+      if Util.testing?
+        return issues[0]
+      end
+
+      highline.choose do |menu|
+        menu.header = 'Choose an issue'
+
+        issues.each do |issue|
+          state = issue['state']['name']
+          if issue['assignee']
+            assignee = issue['assignee']['name']
+            assignee_is_me = issue['assignee']['isMe']
+          else
+            assignee = "none"
+            assignee_is_me = false
+          end
+
+          if state == 'In Progress'
+            state = HighLine::GREEN + state + HighLine::RESET
+          else
+            state = HighLine::RED + state + HighLine::RESET
+          end
+
+          state += HighLine::BOLD if assignee_is_me
+
+          label = "(#{assignee}, #{state}) #{issue['title']}"
+          label = bold(label) if assignee_is_me
+
+          menu.choice(label) { return issue }
+        end
+
+        menu.hidden ''
+        print reset_loading_message # Once menu is build
+      end
+
+      nil # Return nothing
+    end
+
+    def create_commit(title, description, *git_args)
+      extra = highline.ask("\nAdd an optional message").strip
+      title << ' - ' << extra if extra != ''
+      Util.run!(['git', 'commit', '--allow-empty', '-m', title, '-m', description, *git_args])
+    end
+
+    def applicable_issues
+      if Util.testing?
+        settings.linear_api_key
+        return ENV['GEORDI_TESTING_NO_LINEAR_ISSUES'] == 'true' ? [] : [{
+                                                                          'identifier' => '12',
+                                                                          'title' => 'Test Issue',
+                                                                          'url' => 'https://www.issue-url.com',
+                                                                          'branchName' => 'testuser/12-test-issue'
+                                                                        }]
+      end
+
+      team_ids = settings.linear_team_ids
+      filter = {
+        "team": {
+          "id": {
+            "in": team_ids,
+          }
+        },
+        "state": {
+          "type": {
+            "eq": "started"
+          }
+        }
+      }
+      response = request_with_payload(<<~GRAPHQL, filter: filter)
+        query Issues($filter: IssueFilter) {
+          issues(filter: $filter) {
+            nodes {
+              title
+              identifier
+              url
+              branchName
+              assignee {
+                name
+                isMe
+              }
+              state {
+               name
+             }
+            }
+          }
+        }
+      GRAPHQL
+
+      response.dig(*%w[issues nodes])
+
+    end
+
+    def request_with_payload(attributes, variables)
```

Comment:
Please be more specific about what this method does:
```suggestion
    def query_api(attributes, variables)
```